protocol/0027-ASSP-asset_proposal.md

```diff
@@ -163,18 +166,21 @@ This must be an integer strictly greater than `0`.
 - `quantum` is a required parameter  (<a name="0027-ASSP-004" href="#0027-ASSP-004">0027-ASSP-004</a>)
 - If an update asset proposal passed and it changes `quantum` _only_ then this new value becomes used immediately on enactment. (<a name="0027-ASSP-025" href="#0027-ASSP-025">0027-ASSP-025</a>)
+- As a user if I submit a new proposal asset for an ERC20 asset without specifying a chain-id, the proposal is rejected (<a name="0027-ASSP-031" href="#0027-ASSP-031">0027-ASSP-031</a>)
+- As a user if I submit a new proposal asset for an ERC20 asset with a chain-id that does not correspond to any bridge, the proposal is rejected (<a name="0027-ASSP-032" href="#0027-ASSP-032">0027-ASSP-032</a>)
+- As a user if, I can see the chain-id an ERC20 asset originates on via a data node API (<a name="0027-ASSP-033" href="#0027-ASSP-033">0027-ASSP-033</a>)
 
 ### Node actions
 
-- As a node, when a new asset proposal is emitted, I can validate the asset with it's chain, and send the result of the validation through the chain to the other nodes (first phase proposal) (<a name="0027-ASSP-005" href="#0027-ASSP-005">0027-ASSP-005</a>)
-- As a node, when a new asset is accepted through governance, I can sign a payload to the user so they can whitelist the asset with the bridge (<a name="0027-ASSP-006" href="#0027-ASSP-006">0027-ASSP-006</a>)
-- As a node, I receive events from the external blockchain queue, that confirm the asset is enabled in the bridge. (<a name="0027-ASSP-007" href="#0027-ASSP-007">0027-ASSP-007</a>)
-- As a node, when an existing asset is modified through governance changing any one of `maximumLifetimeDeposit` or `withdrawalDelayThreshold`, emit a signed a payload to the world so that they can update the corresponding parameters on the bridge (<a name="0027-ASSP-008" href="#0027-ASSP-008">0027-ASSP-008</a>)
+- As a node, when a new asset proposal is emitted, I can validate the asset with it's chain, and send the result of the validation through the chain to the other nodes (first phase proposal) on any asset bridge  (<a name="0027-ASSP-027" href="#0027-ASSP-027">0027-ASSP-027</a>)
+- As a node, when a new asset is accepted through governance, I can sign a payload to the user so they can whitelist the asset on any asset bridge (<a name="0027-ASSP-028" href="#0027-ASSP-028">0027-ASSP-028</a>)
+- As a node, I receive events from the external blockchain queue, that confirm the asset is enabled on any asset bridge (<a name="0027-ASSP-029" href="#0027-ASSP-029">0027-ASSP-029</a>)
+- As a node, when an existing asset is modified through governance changing any one of `maximumLifetimeDeposit` or `withdrawalDelayThreshold`, emit a signed a payload to the world so that they can update the corresponding parameters on any asset bridge (<a name="0027-ASSP-030" href="#0027-ASSP-030">0027-ASSP-030</a>)
```

Comment:
```suggestion
- As a node, when an existing asset is modified through governance changing any one of `maximumLifetimeDeposit` or `withdrawalDelayThreshold`, emit a signed a payload to the world so that they can update the corresponding parameters on the EVM/ERPC chain the asset is bridged from (<a name="0027-ASSP-030" href="#0027-ASSP-030">0027-ASSP-030</a>)
```